### DISCUSSION TITLE
Two separate Nominatim databases on same server

### DISCUSSION THREAD

**Discussion by raynerhoward** [osm-search/Nominatim#2340] [Q&A] [answered]
I have a Nominatim database set up for a small portion of SE Tennessee and NW Georgia, USA.  Addresses can be searched using only house number and street name.  No need to enter the city, state, zip code or country.  It searches OSM data for teh address and falls back to Tiger 2020 data if unable to find a match.  Geocoding and reverse geocoding are working as well as can be expected.

On the same server, for a completely separate application, I would like to add a single European country.  Or, perhaps a few European countries that border each other.  Is it possible to create a separate nominatim project that points to a separate nominatim database?  I don't want to spin up another VPS for this separate area if I can help it.  I also don't want to combine the two areas using import_multiple_regions.sh since I'm afraid searches will no longer work without adding the city, state, zip code or country.  Using the latest version 3.7.1.

**Comment by lonvia** [comment, marked as answer]
There is no problem to have two separate Nominatim installations on the same machine. Create a second project directory and add a custom configuration `.env` where `NOMINATIM_DATABASE_DSN` points to a different database. Then do the import as usual using the European country excerpt. On the webserver site you need to point the two endpoints to the website directory of the respective project directory and you are set.
